src/models/driver.py

```diff
@@ -0,0 +1,29 @@
+import rethinkdb as r
+from remodel.models import Model
+from remodel.helpers import create_tables, create_indexes
+
+# import classes within the same directory
+from course import Course
+from section import Section
+
+
+def main():
+    conn = r.connect(host='localhost', port=28015, db='testdb')
```

Comment:
This shouldn't be hardcoded in the future, can we put an arg in the makefile where the DB is running?